File: Loop/For_loop/prime.py
n=int(input("enter number"))
prime_is= True
for i in range(2, int(n ** 0.5)+1):
    if(n%i==0): 
       prime_is= False
       break
if prime_is and n>1:
    print(n,"number is prime")
else:
    print(n,"is not prime")

File: Loop/For_loop/prime.java
import java.util.*;
  class Prime{
  public static void main(String args[]){
      int i,n,count=0;
      Scanner sc= new Scanner(System.in);
      n = sc.nextInt();
      for(i=1;i<n;i++){
      if(n%i==0){
      count++;
      sc.close();
  }
 }
      if( count==2){
      System.out.println("number is prime");
  }
      else {
      System.out.println("number is not prime");
  }
 }
}